db/migrate/20190829094432_add_first_middle_last_names_to_claim.rb

```diff
@@ -0,0 +1,7 @@
+class AddFirstMiddleLastNamesToClaim < ActiveRecord::Migration[5.2]
+  def change
+    add_column :claims, :first_name, :string, limit: 100
```

Comment:
How were these limits chosen?

Reply:
I just halved the full name one, happy to change it though

Reply:
no, It should be fine.